### PR TITLE
Escaping html characters in logs view page

### LINKED ISSUE #560: Dupe logs in Warning view
Branch/Commit: Develop
What happened: Logs from backlog showing "dupped" in warning view
What you expected: To only show 1 of the same in warning view
Small snipped from warning logs, with only important dupe logs:
```
2016-05-09 12:36:10 SEARCHQUEUE-BACKLOG-78904 :: [Demonoid] :: [55c0658] Unable to find episode with date 0005-02-16 for show Australian Story, skipping
2016-05-09 12:28:43 SEARCHQUEUE-BACKLOG-287254 :: [Rarbg] :: [55c0658] Cant find search_tvdb in database. Are you sure it exists?
2016-05-09 12:23:34 SEARCHQUEUE-BACKLOG-278798 :: [Rarbg] :: [55c0658] Cant find search_tvdb in database. Are you sure it exists?
2016-05-09 12:08:11 SEARCHQUEUE-BACKLOG-254184 :: [Rarbg] :: [55c0658] Cant find search_tvdb in database. Are you sure it exists?
2016-05-09 12:06:05 SEARCHQUEUE-BACKLOG-254184 :: [Demonoid] :: [55c0658] Unable to find episode with date 0005-02-16 for show Australian Story, skipping
2016-05-09 11:51:25 SEARCHQUEUE-BACKLOG-277096 :: [Rarbg] :: [55c0658] Cant find search_tvdb in database. Are you sure it exists?
2016-05-09 11:28:54 SEARCHQUEUE-BACKLOG-110381 :: [Demonoid] :: [55c0658] Unable to find episode with date 0005-02-16 for show Australian Story, skipping
2016-05-09 10:47:37 SEARCHQUEUE-BACKLOG-82186 :: [Rarbg] :: [55c0658] Cant find search_tvdb in database. Are you sure it exists?
2016-05-09 10:42:26 SEARCHQUEUE-BACKLOG-82186 :: [Demonoid] :: [55c0658] Unable to find episode with date 0005-02-16 for show Australian Story, skipping
2016-05-09 10:26:42 SEARCHQUEUE-BACKLOG-83982 :: [Rarbg] :: [55c0658] Cant find search_tvdb in database. Are you sure it exists?
2016-05-09 10:06:53 SEARCHQUEUE-BACKLOG-100741 :: [Rarbg] :: [55c0658] Cant find search_tvdb in database. Are you sure it exists?
2016-05-09 09:59:06 SEARCHQUEUE-BACKLOG-100741 :: [Demonoid] :: [55c0658] Unable to find episode with date 0005-02-16 for show Australian Story, skipping
2016-05-09 09:56:27 SEARCHQUEUE-BACKLOG-281341 :: [Rarbg] :: [55c0658] Cant find search_tvdb in database. Are you sure it exists?
2016-05-09 09:44:10 SEARCHQUEUE-BACKLOG-271546 :: [Rarbg] :: [55c0658] Cant find search_tvdb in database. Are you sure it exists?
2016-05-09 09:40:27 SEARCHQUEUE-BACKLOG-273859 :: [Rarbg] :: [55c0658] Cant find search_tvdb in database. Are you sure it exists?
2016-05-09 09:32:42 SEARCHQUEUE-BACKLOG-263546 :: [Rarbg] :: [55c0658] Cant find search_tvdb in database. Are you sure it exists?
2016-05-09 09:28:50 SEARCHQUEUE-BACKLOG-272876 :: [Rarbg] :: [55c0658] Cant find search_tvdb in database. Are you sure it exists?
2016-05-09 09:20:04 SEARCHQUEUE-BACKLOG-292136 :: [Rarbg] :: [55c0658] Cant find search_tvdb in database. Are you sure it exists?
2016-05-09 09:13:24 SEARCHQUEUE-BACKLOG-294117 :: [Rarbg] :: [55c0658] Cant find search_tvdb in database. Are you sure it exists?
2016-05-09 08:50:12 SEARCHQUEUE-BACKLOG-268592 :: [Demonoid] :: [55c0658] Unable to find episode with date 0005-02-16 for show Australian Story, skipping
2016-05-09 08:38:27 SEARCHQUEUE-BACKLOG-280473 :: [Rarbg] :: [55c0658] Cant find search_tvdb in database. Are you sure it exists?
2016-05-09 08:07:33 SEARCHQUEUE-BACKLOG-259765 :: [Demonoid] :: [55c0658] Unable to find episode with date 0005-02-16 for show Australian Story, skipping
2016-05-09 08:02:15 SEARCHQUEUE-BACKLOG-77904 :: [Rarbg] :: [55c0658] Cant find search_tvdb in database. Are you sure it exists?
2016-05-09 07:39:48 SEARCHQUEUE-BACKLOG-72514 :: [Rarbg] :: [55c0658] Cant find search_tvdb in database. Are you sure it exists?
2016-05-09 07:32:17 SEARCHQUEUE-BACKLOG-247897 :: [Demonoid] :: [55c0658] Unable to find episode with date 0005-02-16 for show Australian Story, skipping
2016-05-09 07:16:43 SEARCHQUEUE-BACKLOG-77466 :: [Rarbg] :: [55c0658] Cant find search_tvdb in database. Are you sure it exists?
```

### PR DESCRIPTION
**Pull request by ratoaq2**
- [x] PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review
- [x] Read [contribution guide](https://github.com/PyMedusa/SickRage/blob/master/.github/CONTRIBUTING.md)

- Escaping html characters in logs view page: When the log message contains < > all the information inside brackets are not displayed. This should fix #560 
- Removing errors_logs.py from flake8 whitelist.
- Making it more efficient (precompile log regex and other minor enhancements)

### CODE REVIEW

File: sickbeard/server/web/core/error_logs.py [hunk 1]
```diff
@@ -1,172 +1,177 @@
...
 from sickrage.helper.encoding import ek
-from sickbeard.server.web.core.base import WebRoot, PageTemplate
+from tornado.routes import route
+
+
+# log regular expression: 2016-08-06 15:58:34 ERROR    DAILYSEARCHER :: [d4ea5af] Exception generated in thread DAILYSEARCHER
+log_re = re.compile(r'^(?P<year>\d{4})-(?P<month>\d{2})-(?P<day>\d{2})\s+'
+                    r'(?P<hour>\d{2}):(?P<minute>\d{2}):(?P<second>\d{2})\s+'
+                    r'(?P<log_level>[A-Z]+)\s+(?P<log_name>.+?)\s+::\s+(?P<log_message>.*)$')
+
```
Comment (fernandog) [inline sickbeard/server/web/core/error_logs.py]: When the error is in the providers, the name of the  provider appear in the log message too

Also there is the numbers next to thread name like BACKLOGSEARCH-7385828

Comment (ratoaq2) [inline sickbeard/server/web/core/error_logs.py]: I just moved the old regex and added names to the capturing groups. You're saying that the current implementation has that issue as well?

Comment (fernandog) [inline sickbeard/server/web/core/error_logs.py]: I think so.

Comment (ratoaq2) [inline sickbeard/server/web/core/error_logs.py]: I just saw and tested it... the numbers are captured correctly here:
`(?P<log_name>.+?)`

Comment (ratoaq2) [inline sickbeard/server/web/core/error_logs.py]: and the commit hash and provider is in the rest:
`(?P<log_message>.*)`

**Comment by fernandog** [comment]
Approved

**Pull request merged**